can_tools/scrapers/official/WI/wi_state.py

```diff
@@ -5,12 +5,11 @@
 from can_tools.scrapers.official.base import ArcGIS
 
 
-class Wisconsin(ArcGIS, DatasetBase):
+class WisconsinCounties(ArcGIS, DatasetBase):
```

Comment:
@MattSidor @sglyon 

Given that these scrapers are pretty "big", what are our thoughts on moving them into two separate files?

Reply:
I'd be ok either way.

Reply:
K -- I thought we had started doing more separation of scrapers, but, after looking at some of the others, I think I'm okay leaving these together and just defining the `WisconsinArcGIS` class at the top of the file.